src/Nethermind/Nethermind.Baseline/Tree/BaselineTree.cs

```diff
@@ -193,6 +195,29 @@ public BaselineTreeNode[] GetLeaves(params uint[] leafIndexes)
             return leaves;
         }
 
+        public void RecalculateHashes()
```

Comment:
you will need  method that will take the start leaf index too

Reply:
> you will need method that will take the start leaf index too

I know :) I planned it in next PRs :)

Reply:
@tkstanczak added :)